src/compas_rhino/scene/scene.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import compas_rhino
+from compas_rhino.artists import Artist
+
+
+__all__ = ['Scene', 'SceneNode']
+
+
+class SceneNode(object):
+
+    def __init__(self, scene, item, **kwargs):
+        self.scene = scene
+        self.item = item
+        self.artist = Artist.build(item, **kwargs)
+        self.nodes = []
+
+
+class Scene(object):
+    """Rhino scene object for managing the visualisation of COMPAS data and geometry.
+
+    Notes
+    -----
+    * Add "display mode": `Shaded`, `Ghosted`, ...
+    * Add "projection": `Perspective`, `Orthogonal`, ...
+    * Add "viewport": `Front`, `Left`, `Top`, `Perpsective`
+    * Add "camera": ...
+    """
+
+    def __init__(self):
+        self.nodes = []
+
+    def add(self, item, **kwargs):
+        node = SceneNode(self, item, **kwargs)
+        self.nodes.append(node)
+        return node
+
+    def update(self):
+        compas_rhino.rs.EnableRedraw(False)
+        for node in self.nodes:
+            node.artist.draw()
+        compas_rhino.rs.EnableRedraw(True)
+
+    def save(self, path, width=1920, height=1080, scale=1,
```

Comment:
I am not sure `save` a scene is logically the same as screenshoting the current view. I would expect saving a scene means saving all meshes and scene configuration into some file format (like the glTF things that is in progress)